package.json

```diff
@@ -159,7 +159,7 @@
     "mock-fs": "^4.8.0",
     "mockdate": "^3.0.2",
     "moment": "^2.30.1",
-    "nock": "13.5.6",
+    "nock": "14.0.4",
```

Comment:
If this works, maybe we shouldn't pin to this version?

```suggestion
    "nock": "^14.0.4",
```

Reply:
Sorry, figured this was probably a WIP but put in my initial review just cuz :)

Reply:
no worries.. I also found another way to do what I was planning to do, so I'll just close it for now and create an issue to do upgrade later